src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method name 'getStatistic' could be more descriptive. Consider renaming it to reflect the actions being performed, such as 'generateReportFromDataFile'.